src/main/resources/reference.conf

```diff
@@ -9,6 +9,8 @@ dataproc {
   welderServerName = "welder-server"
   firewallRuleName = "leonardo-notebooks-rule"
   networkTag = "leonardo"
+  projectVPCNetworkLabel = "vpc-network-name"
+  projectVPCSubnetLabel = "vpc-subnetwork-name"
```

Comment:
Do you want this here, or in fc-dev leo.conf?

Reply:
I think here is fine, unless we envision this varying by environment. 

Can you also update `leonardo-example.conf` and add some documentation?